public/app/features/dashboard-scene/saving/DashboardPrompt.tsx

```diff
@@ -99,6 +100,24 @@ export const DashboardPrompt = memo(({ dashboard }: DashboardPromptProps) => {
       },
 
       onDiscard: () => {
+        // Track dashboard library discard if this is a library dashboard
+        if (originalPath === DASHBOARD_LIBRARY_ROUTES.Template) {
```

Comment:
we decided to discard this tracking event, as we can know the number of clicks on the item and subtract with the number of saves. However, I like this as it contains more data and we could maybe know the most discarded dashboards in a better way? 
Can you add these tracking events in [this table](https://docs.google.com/document/d/1DopWOTkFO-ZqueNezZ7fV_m0fcuDIrIT-W4KhRVwZmM/edit?tab=t.jhq97meo8j3h#heading=h.ih1y6obfplyx)?
(you can check that `grafana_dashboard_prompt_discard_template_dashboard` was discarded)
Thoughts? @nmarrs